Constitution.md

```diff
@@ -122,21 +121,18 @@ If there is more then 1 president, they exist as a single voting member on counc
 15. Enforce that the transition report is done by April 30 of the council’s term.
 16. Perform any other duties incidental to the office of the President or directed by Council.
 
-### Director of Finances and Internal Affairs
-
-1. Act as a signing authority for CSSA.
-2. Providing a brief report on CSSA finances and expenses at each meeting.
-3. Be responsible for all funds generated by Council events and for overseeing all funds spent by Council.
-4. Make available all audits of CSSA to any Student members upon request with more than two weeks of notice.
-5. Ensure the Council is kept updated on ongoing council initiatives.
-6. Perform a year-end audit at the end of the CSSA Fiscal Year.
-7. Reserve the meeting room for CSSA meetings, General meetings and the CS Executive Assembly.
-8. Prepare an agenda and council package for Regular Meetings and distribute them to the Council at least two days prior to the meeting.
-9. Receive resignations of Council and inform Executives and the Computer Science student body of any such resignations.
-10. Ensure that the CSSA lounge is a safe and inclusive space.
-11. Lead the Financial Accounability Committee with up to 3 committee members.
-12. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+### Director of Internal Affairs
 
+1. Ensure the Council is kept updated on ongoing council initiatives.
+2. Reserve the meeting room for CSSA executive meetings, general meetings and the CS Executive Assembly.
+3. Prepare an agenda and council package for executive meetings and distribute them to the Council at least one week prior to the meeting.
+4. Record meeting minutes during CSSA executive meetings and CS Executive Assemblies, or appoint another Council member to do so.
+5. Work with the President(s) to assign any actionable tasks that come about in meetings to members of Council.
+6. Work with the Director of Events to reserve rooms, equipment, and/or catering for on-campus activites.
+7. Monitor applications for Councillor positions and present them for consideration at CSSA executive meetings.
```

Comment:
Is this internal's job? should it just be done by the leads of the relevant committee?

Reply:
Internal is making sure of it. They can choose to delegate to others if they want

Reply:
Councillor apps can be independent of committees in my mind's eye so used Internal to cover that case. Ultimately if a committee lead decides they want to do the monitoring/presenting/etc for their own committee, that's no problem! Internal should be the one taking the reponsibility if nobody else volunteers though, agree